core-primitives/utils/Cargo.toml

```diff
@@ -11,6 +11,7 @@ hex = { version = "0.4.3", default-features = false, features = ["alloc"] }
 
 # substrate
 frame-support = { default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
+sp-core = { default-features = false, features = ["full_crypto"], git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.20" }
```

Comment:
can you add this to the std feature?

Reply:
👍 glad you have such a keen eye to always spot these 😄